### ISSUE TITLE
Bind to something else than localhost

### ISSUE THREAD

**Issue by juRiii** [aspnet/Home#799]
Hi,

im using this `project.json`:

	"commands": {
		"web": "Microsoft.AspNet.Hosting --config hosting.ini"
	},

with the following `hosting.ini`:

	server=Microsoft.AspNet.Server.WebListener
	server.urls=http://localhost:5000

I tried to access `http://myip.local:5000/api/values`

but im getting:

	Bad Request - Invalid Hostname

	HTTP Error 400. The request hostname is invalid.
	
only `http://localhost:5000/api/values` is working

suggestions?

**Comment by juRiii** [comment]
One more thing: I dont want to use IIS Express

**Comment by Eilon** [comment]
@Tratcher can you take a look?

**Comment by Tratcher** [comment]
Change `server.urls=http://localhost:5000` to `server.urls=http://*:5000`, but if you do that you'll need to run the process (or VS) as admin.

Here's a write-up for configuring Http.Sys:
https://katanaproject.codeplex.com/wikipage?title=Selfhosting

**Comment by juRiii** [comment]
thank you, its working now.

**Comment by stantoxt** [comment]
That's Greate! I am wondering when i start the server with kestrel,It works both without change Change server.urls=http://localhost:5000 to server.urls=http://*:5000.

**Comment by Tratcher** [comment]
Kestrel uses raw sockets and only looks at the port (currently), so localhost vs * doesn't make a difference. That will change. https://github.com/aspnet/KestrelHttpServer/issues/98

**Comment by stantoxt** [comment]
Tratcher,Thanks.I've learned a lot.

**Comment by robie01** [comment]
Hi i've build project in angular but couldn't see the changes in my browser. I have tried to debug and I got this error: Pls anyone can help me?
Chrome 61.0.3163 (Mac OS X 10.12.5): Executed 0 of 4 SUCCESS (0 secs / 0 secs)
Chrome 61.0.3163 (Mac OS X 10.12.5) ERROR
  Uncaught NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'ng:///DynamicTestModule/VideoComponent.ngfactory.js'.
  at http://localhost:9876/_karma_webpack_/polyfills.bundle.js:2736
Chrome 61.0.3163 (Mac OS X 10.12.5) ERROR
  Uncaught NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'ng:///DynamicTestModule/VideoComponent.ngfactory.js'.
  at http://localhost:9876/_karma_webpack_/polyfills.bundle.js:2736
Chrome 61.0.3163 (Mac OS X 10.12.5): Executed 0 of 4 ERROR (0 secs / 0 secs)
Chrome 61.0.3163 (Mac OS X 10.12.5) ERROR
  Uncaught NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'ng:///DynamicTestModule/VideoComponent.ngfactory.js'.
  at http://localhost:9876/_karma_webpack_/polyfills.bundle.js:2736
Chrome 61.0.3163 (Mac OS X 10.12.5): Executed 0 of 4 ERROR (0.106 secs / 0 secs)
30 09 2017 22:07:53.797:INFO [Chrome 61.0.3163 (Mac OS X 10.12.5)]: Connected on socket X3F_mCPDoS6rRil5AAAB with id manual-4403
Chrome 61.0.3163 (Mac OS X 10.12.5): Executed 0 of 4 SUCCESS (0 secs / 0 secs)
